src/Users/README.md

```diff
@@ -61,16 +61,63 @@ if (!$result) {
 }
 ```
 
+### UPDATE OWN BASIC INFO
+```php
+$updatedUser = \ATDev\RocketChat\Users\User::updateOwnBasicInfo([
+    'email' => 'test@updated.com',
+    'name' => 'Updated Name',
+    'username' => 'new-username',
+    'currentPassword' => hash('sha256', 'q1w2e3r4t5'),
+    'newPassword' => 'newPassw0rd',
+    'customFields' => [
+        'twitter' => '@example'
+    ]
+]);
+if (!$updatedUser) {
+    $error = \ATDev\RocketChat\Users\User::getError();
+}
+```
+
+### SET ACTIVE STATUS
+```php
+$user = new \ATDev\RocketChat\Users\User("[USER ID]");
+$statusResult = $user->setActiveStatus(true);
+if (!$statusResult) {
+    $statusResult->getError();
+}
+```
+
+### SET STATUS MESSAGE
+```php
+$result = \ATDev\RocketChat\Users\User::setStatus("New status message", "away");
```

Comment:
Same as above. Can we do ::me call and then to non-static call on received object?